tools/cudaq-qpud/RestServerMain.cpp

```diff
@@ -31,7 +31,22 @@ int main(int argc, char **argv) {
   llvm::cl::ParseCommandLineOptions(argc, argv, "CUDA Quantum REST server\n");
   if (cudaq::mpi::available())
     cudaq::mpi::initialize();
-  auto restServer = cudaq::registry::get<cudaq::RemoteRuntimeServer>("rest");
+  std::string serverType = "rest";
+  // Check environment variable for any specific server subtype.
+  if (auto serverSubType = std::getenv("CUDAQ_SERVER_TYPE")) {
+    if (cudaq::registry::isRegistered<cudaq::RemoteRuntimeServer>(
+            serverSubType)) {
+      printf("[cudaq-qpud] Using server subtype: %s\n", serverSubType);
```

Comment:
When this is deployed, will we be able to see this message to verify that we have correctly set it up? I'm wondering if it might be more clear if we use regular command-line parsing here vs environment variables. What do you think?

Reply:
Yes, we should be able to see this log in NVCF. I think it makes more sense to use regular command-line parsing. Will push an update shortly.

Reply:
Fixed in https://github.com/NVIDIA/cuda-quantum/pull/1210/commits/27e5eb8bd44591d4c71fc02c73c41cfa7e4b9fab along with `CallOp` verification. 
We allow calls to QIR functions and a couple of registered NVQIR functions.